<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstrumentoDialog</class>
 <widget class="QewTreeViewDialog" name="InstrumentoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="containerName">
   <cstring>stackedWidget</cstring>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="datosInstrumento">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Id"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Selecciona el intervalo</string>
         </property>
         <widget class="QRadioButton" name="radioButton_0127">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>32</y>
            <width>57</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>0-127</string>
          </property>
         </widget>
         <widget class="QRadioButton" name="radioButton_1128">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>61</y>
            <width>57</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>1-128</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_Banco">
           <property name="text">
            <string>AÃ±adir Banco</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QewExtensibleDialog</class>
   <extends>QDialog</extends>
   <header>qewextensibledialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QewTreeViewDialog</class>
   <extends>QewExtensibleDialog</extends>
   <header>qewtreeviewdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
